test/specs/modules/Dropdown/Dropdown-test.js

```diff
@@ -620,6 +620,62 @@ describe('Dropdown Component', () => {
     })
   })
 
+  describe('removing items on backspace', () => {
+    let spy
+    beforeEach(() => {
+      spy = sandbox.spy()
+    })
+
+    it('does nothing without selected items', () => {
+      wrapperMount(<Dropdown {...requiredProps} multiple search onChange={spy} />)
+
+      // open
+      wrapper.simulate('click')
+
+      domEvent.keyDown(document, { key: 'Backspace' })
+
+      spy.should.not.have.been.called()
+    })
+    it('removes the last item when there is no search query', () => {
+      const value = _.map(options, 'value')
+      const expected = _.without(value, _.last(value))
+      wrapperMount(<Dropdown {...requiredProps} value={value} multiple search onChange={spy} />)
+
+      // open
+      wrapper.simulate('click')
+
+      domEvent.keyDown(document, { key: 'Backspace' })
+
+      spy.should.have.been.calledOnce()
+      spy.firstCall.args[1].should.deep.equal(expected)
```

Comment:
I'll need to add a test with an uncontrolled component. However, I'm having an issue where even though `expected` and `state.value` have the same elements, the assertion fails with `expected <Dropdown /> to have a 'value' state with the value [ Array(4) ], but the value was [ Array(4) ]`. Is there something other than `should.have.state` that I should be using instead? Looking through other tests in Stardust, I don't see any usable of `should.have.state` with arrays.